src/main/java/teammates/common/datatransfer/questions/FeedbackContributionResponseDetails.java

```diff
@@ -202,14 +202,13 @@ private String getContributionQuestionResponseAnswerCsv(
 
     public Map<String, TeamEvalResult> getContribQnTeamEvalResult(FeedbackQuestionAttributes question,
             FeedbackSessionResultsBundle feedbackSessionResultsBundle) {
-        Map<String, TeamEvalResult> contribQnStats =
-                feedbackSessionResultsBundle.contributionQuestionTeamEvalResults.get(question.getId());
-        if (contribQnStats == null) {
+        feedbackSessionResultsBundle.contributionQuestionTeamEvalResults.computeIfAbsent(question.getId(), key -> {
+            Map<String, TeamEvalResult> contribQnStats =
+                    feedbackSessionResultsBundle.contributionQuestionTeamEvalResults.get(question.getId());
             FeedbackContributionQuestionDetails fqcd = (FeedbackContributionQuestionDetails) question.getQuestionDetails();
             contribQnStats = fqcd.getTeamEvalResults(feedbackSessionResultsBundle, question);
-            feedbackSessionResultsBundle.contributionQuestionTeamEvalResults.put(question.getId(), contribQnStats);
-        }
-
-        return contribQnStats;
+            return contribQnStats;
+        });
+        return feedbackSessionResultsBundle.contributionQuestionTeamEvalResults.get(question.getId());
```

Comment:
I think you can save the getter method by setting a variable at `feedbackSessionResultsBundle.contributionQuestionTeamEvalResults.computeIfAbsent` as `computeIfAbsent` will return the default value.